conf.d/main

```diff
@@ -6,12 +6,12 @@ DB_PASS=$(mcookie)
 
 SRC=/usr/local/src
 WEBROOT=/var/www/opencart
-STORAGE=$(dirname $WEBROOT)/opencart-storage
+STORAGE=/var/www/opencart-storage
```

Comment:
Curious why you made this change? It's no big deal and don't bother changing it now, but they should be exactly the same path!? It's just one is static and one is generated dynamically (so `STORAGE` will always sit beside `WEBROOT`).